src/ae/index.js

```diff
@@ -188,7 +188,8 @@ const Ae = stampit(NodePool, Tx, AccountResolver, ContractCompilerHttp, {
             onAccount: this,
             onCompiler: {
               compilerApi: this.compilerApi,
-              compilerVersion: this.compilerVersion
+              compilerVersion: this.compilerVersion,
+              api: this.api
             }
```

Comment:
```suggestion
            onCompiler: this.compilerApi
```
`compilerVersion` not used as I see, `api` provided by `onNode`, so this should be enough